State.h

```diff
@@ -0,0 +1,82 @@
+#ifndef STATE_H
+#define STATE_H
+
+#include "BreathData.h"
+
+/*
+ *  Parent class for all instantiated state classes
+ */
+class State {
+  public:
+    virtual State* update() = 0;           // perform maintenance, and perhaps transition to new state
+    static BreathData* breath;             // holds shared information about the breath cycle
+};
+
+/*
+ *  Specific states used by the ventilator control.
+ *  Only a singleton instance of each is created, accessible via the get() method.
+ */
+
+// Standby mode.  This is the initial state.
+class OffState : public State {
+  public:
+    static State* enter();               // performs actions necessary upon entering the state
+    State* update();                     // perform maintenance, and perhaps transition to next state
+    static State* begin(BreathData);     // returns the OffState instance with breath data link
+  private: 
+    OffState() {};                       // to create singleton instance
+    static OffState *instance;           // storage for singleton instance
```

Comment:
This should be unnecessary.  See my comment for `OffState::begin` in `State.cpp`.

Reply:
At the very least, it should be an object, not a pointer, as I described in State.cpp.

Reply:
Fixed.